spec/requests/meetings/google_calendar_event_spec.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
-RSpec.describe ::Meetings::GoogleCalendarEventController, type: :controller do
-  include_context(:logged_in_user)
+RSpec.describe 'GoogleCalendarEvent', type: :request do
   let(:user) { create(:user_oauth) }
   let(:meeting) { create(:meeting) }
   let(:calendar_uploader) { double }
-  let!(:calendar_event) { double(id: 'someid') }
-  let!(:exception_message) { 'Exception message' }
+  let(:calendar_event) { double(id: 'someid') }
+  let(:exception_message) { 'Exception message' }
```

Comment:
Not sure if I'm missing anything here, but I don't think this needs to be `let!` and just `let` seems to be enough. The test still passes this way and there doesn't seem to be any model here that needs to be created and initialize before each test. 

On the other hand, `let!(:meeting_member)` below is required since meeting_member needs to be created before the **act** step to be altered after. If we don't use `let!` there, `meeting_member` will be created during the **assert** step `expect(meeting_member.reload.google_cal_event_id).to eq(calendar_event.id)` and would've have never been altered to meet the proper assertion.

Reply:
Good catch, thanks so much for cleaning this up!